lib/commandlineio.rb

```diff
@@ -0,0 +1,48 @@
+require_relative "blackjack_score"
+
```

Comment:
you could use `require` rather than `require relative` here

maybe you could research the difference - might make a good blog post